src/Bridge/Doctrine/Orm/Extension/EagerLoadingExtension.php

```diff
@@ -25,22 +25,23 @@
  * @author Charles Sarrazin <charles@sarraz.in>
  * @author Kévin Dunglas <dunglas@gmail.com>
  * @author Antoine Bluchet <soyuka@gmail.com>
+ * @author Baptiste Meyer <baptiste.meyer@gmail.com>
```

Comment:
@api-platform/core-team Perhaps we need to have a stricter policy for this. Otherwise there will be unbounded growth. :smile: